blog/2021-10-11-defining-workflows.md

```diff
@@ -0,0 +1,165 @@
+---
+slug: defining-workflows
+title: Defining Workflows
+author: Tihomir Surdilovic
+author_title: Developer Advocate
+author_image_url: https://avatars.githubusercontent.com/u/119422?v=4
+tags:
+  - workflow
+  - community
+  - temporal
+date: 2021-10-11T00:00:00Z
```

Comment:
I would also add an image `image: <url>` or the preview will be a giant picture of your face